src/devices/inovalley-kw9015b.c

```diff
@@ -37,18 +38,34 @@ static int kw9015b_callback(bitbuffer_t *bitbuffer) {
 				(reverse8(bb[i][2])>>4)+(reverse8(bb[i][2])&0x0F)+
 				(reverse8(bb[i][3])>>4)+(reverse8(bb[i][3])&0x0F));
 
+
+                        if (debug_output >= 1) {
+					fprintf(stdout, "\nSensor        = Inovalley kw9015b, TFA Dostmann 30.3161 (Rain and temperature sensor)\n");
+					fprintf(stdout, "Device        = %d\n", device);
+					fprintf(stdout, "Temp          = %f\n",fTemp);
+					fprintf(stdout, "Rain          = %d\n",iRain);
+					fprintf(stdout, "checksum      = %02x==%02x\n",chksum&0xF,reverse8(bb[i][4]));
+					fprintf(stdout, "Received Data = %02X %02X %02X %02X %02X\n",
+			 		reverse8(bb[i][0]),
+					reverse8(bb[i][1]),
+					reverse8(bb[i][2]),
+					reverse8(bb[i][3]),
+					reverse8(bb[i][4]));
+			}
+
 			if( (chksum&0x0F) == ( reverse8(bb[i][4]) &0x0F)){
-				fprintf(stdout, "\nSensor        = Temperature and rain event\n");
-				fprintf(stdout, "Device        = %d\n", device);
-				fprintf(stdout, "Temp          = %f\n",fTemp);
-				fprintf(stdout, "Rain          = %d\n",iRain);
-				fprintf(stdout, "checksum      = %02x==%02x\n",chksum&0xF,reverse8(bb[i][4]));
-				fprintf(stdout, "Received Data = %02X %02X %02X %02X %02X\n",
-				reverse8(bb[i][0]),
-				reverse8(bb[i][1]),
-				reverse8(bb[i][2]),
-				reverse8(bb[i][3]),
-				reverse8(bb[i][4]));
+
+				/* Get time now */
+				local_time_str(0, time_str);
+
+				data = data_make("time", "", DATA_STRING, time_str,
+					"model", "", DATA_STRING, "Inovalley kw9015b, TFA Dostmann 30.3161 (Rain and temperature sensor)",
```

Comment:
The model string looks rather verbose. This string would likely be used as a key for database logging.